src/main/java/executor/command/CommandIncome.java

```diff
@@ -0,0 +1,109 @@
+package executor.command;
+
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.IncomeReceipt;
+import ui.Receipt;
+import ui.Ui;
+import ui.Wallet;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class CommandIncome extends Command {
```

Comment:
Refactor the Class to be CommandAddIncomeReceipt or CommandCreateIncomeReceipt so it's very clear what the Class is used for.